### PR TITLE
miner_tests number clarification

### PR DESCRIPTION
**Pull request by instagibbs**
The unchecked simulated fee is 10x smaller than the actual fee. Doesn't hurt the test, but lacks clarity for the reader.

### CODE REVIEW

**Comment by morcos** [comment]
ah shoot, i thought i double checked all those
thanks

utACK

**Comment by laanwj** [comment]
I'm sorry, but I don't think the new magic value is much clearer than the older one.
Could we format this as an expression? (e.g. if there is a factor *10, make it explicit)

**Comment by MarcoFalke** [comment]
Agree with @laanwj. It is really hard to parse such numbers without putting the finger/cursor on the screen and counting... Using `1000 * 1000 * ...` or `int(4e6)` could make sense, imo.

Maybe even add `MEGA`, `GIGA`, `MEBI`, `GIBI` etc as constants to some header file?

**Comment by instagibbs** [comment]
I can do that sure

**Comment by morcos** [comment]
@laanwj Yeah I hate these things as well.  But the right answer is not to write them more clearly but to get rid of the need to manually calculate them.  They are not magic values but just the mechanical result of subtracting valueOut from valueIn.  They used to just be random values but then mining code was changed to depend on the value in CTxMemPoolEntry instead of calculating it itself, and so I figured it was safer to put the right calculation in there and avoid spurious test errors.  I've been meaning to create a helper class to generate CTxMemPoolEntry's in tests for some time.

**Comment by laanwj** [comment]
> But the right answer is not to write them more clearly but to get rid of the need to manually calculate them.

Computing the values in the test itself is exactly what I mean by 'writing more clearly'. It means it's possible to see where the number comes from.

**Comment by instagibbs** [comment]
>(this is not just idle aesthetics by me, but I know at some point the test will need to be updated, and tons of numbers without clear explanation will make that more difficult)  

Yes, I had to diagram the transactions to figure out what it was trying to do and convince myself they were wrong.
